architecture/core_concepts/routes.adoc

```diff
@@ -12,22 +12,27 @@ toc::[]
 
 == Overview
 
-An OpenShift route is a way to expose a
-link:pods_and_services.html#services[service] by giving it an
-externally-reachable hostname like `www.example.com`.
+An OpenShift link:../architecture/core_concepts/routes.html[route] exposes a
+link:../architecture/core_concepts/pods_and_services.html#services[service] at a
+host name, like _www.example.com_, so that external clients can reach it by
+name.
 
-A defined route and the endpoints identified by its service can be consumed by a
-router to provide named connectivity that allows external clients to reach your
-applications. Each route consists of a route name (limited to 63 characters),
-service selector, and (optionally) security configuration.
+DNS resolution for a host name is handled separately from routing;
+your administrator may have configured a cloud domain that will always
```

Comment:
Cloud domain?  Do you mean a DNS wildcard entry as described here: https://docs.openshift.com/enterprise/3.2/install_config/install/prerequisites.html#prereq-dns